### DISCUSSION TITLE
MicroUI 1/4 screen interface

### DISCUSSION THREAD

**Discussion by peacebro** [Nevysha/Cozy-Nest#170] [Ideas]
Hi there, great work on CozyNest. Someone just showed it to me a few hours ago and its a huge improvement coming from plain WebUI. I had an idea I wanted to share: MicroUI. I think it suits certain users more than others. For cinema related stuff for example, we dont need as much tweaking of settings on the left so it doesnt make sense to have so much screen real estate dedicated to them. Better to be able to focus on what we're creating and not be distracted by anything. Also more useful to have a web browser and notes open, if referencing other images.

Apologies for putting it together sloppily, just wanted to get it out there!

![MicroUI_draft1](https://github.com/Nevysha/Cozy-Nest/assets/11905671/076ce96f-251a-4a82-bd9b-77a0eab997ab)

Having the prompt directly under the image just feels more "focused". Extensions can be accessed by scrolling down. Cinema folks doing visualizations for projects hardly even use the extensions. Maybe ControlNet once in a while. 

The Extra Networks tab can pop out like it already does.
Could probably hide Restore Face, Tiling, HiRes Fix and CFG Scale fix in a submenu and clear up even more space.
Also, vertical space in the output window isnt super necessary for cinema stuff but if you wanted to make sure folks could still make vertical images, things could still be pushed up and down, theres plenty of space remaining.

I know its a long shot but I just wanted to share. If you ever decide to make a UI like this, or a mode in CozyNest, I'd be super happy to support somehow, like, moneywise if it helped.
Thanks!

**Comment by Nevysha** [comment]
Thanks for those idea. I'm considering them but don't expect anything shortly because that's a lots of work and I don't know atm if I'll implement them.

**Reply by peacebro** [reply]
I totally understand. Also I know its not something most people would even consider. I think MicroUI could be promising in the future when more people use SD. I've been using CozyNest all day today and I'm blown away by how much my workflow has improved already.  Excellent work.
